src/task2/Uniq.java

```diff
@@ -0,0 +1,37 @@
+package task2;
+
+import java.util.Scanner;
+
+public class Uniq {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.print("Введите строку: ");
+        String input = scanner.nextLine();
+
+        String[] words = input.split(" ");
+
+        String minUniqueWord = null;
+        int minUniqueChars = 1000000;
```

Comment:
It's better to avoid such "magic numbers". Either move to constants or use standart Integer.MAX_INT